sdk/compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -35,10 +35,10 @@ main = withSdkVersions $ do
     damlcLegacy <- locateRunfiles ("damlc_legacy" </> exe "damlc_legacy")
     let validate dar = callProcessSilent damlc ["validate-dar", dar]
     v1TestArgs <- do
-        let targetDevVersion = LF.version1_dev
+        let targetDevVersion = LF.version1_17
         let exceptionsVersion = minExceptionVersion LF.V1
         let simpleDalfLfVersion = LF.defaultOrLatestStable LF.V1
-        scriptDevDar <- locateRunfiles (mainWorkspace </> "daml-script" </> "daml" </> "daml-script-1.dev.dar")
+        scriptDevDar <- locateRunfiles (mainWorkspace </> "daml-script" </> "daml" </> "daml-script-1.17.dar")
```

Comment:
Theres a few places like this one where we're pulling daml-script-1.17 into a variable that says it's dev.